src/index.ts

```diff
@@ -0,0 +1,14 @@
+import express from "express";
+const app = express();
+const port = 8080; // default port to listen
```

Comment:
ubernit: If you want, you can add an env var for this and leave 8080 as a default

Reply:
Will add proper cmd line arguements in a later PR.